src/routes/groupAssociationsRoutes.js

```diff
@@ -7,8 +7,8 @@ const accessControl = require('../middleware/acl');
 const associationsRouter = express.Router();
 
 // post, put, and delete for both admin and users for wishlists
-associationsRouter.post('/associations/:groupid/:userid',accessControl('delete'), handleAssociationCreate);
```

Comment:
removed role access control middleware from routes until we incorporate authentication middleware.